### PR TITLE
sys-cluster/torque: bump to 6.1.3

### PR DESCRIPTION
**Pull request by scardracs**
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Marco Scardovi <marco@scardovi.com>

### CODE REVIEW

File: sys-cluster/torque/files/torque-6.1.3-bool-fix.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+diff -up torque/src/drmaa/src/compat.h.bool-fix torque/src/drmaa/src/compat.h
```
Comment (thesamesam) [inline sys-cluster/torque/files/torque-6.1.3-bool-fix.patch:1]: Please link to where patches are from (and any relevant Gentoo bugs too).

File: sys-cluster/torque/files/torque-6.1.3-munge-size.patch [hunk 2]
```diff
@@ -0,0 +1,25 @@
+From 12a8d7dde1d07aed670f0dd50b317b256daaa991 Mon Sep 17 00:00:00 2001
```
Comment (thesamesam) [inline sys-cluster/torque/files/torque-6.1.3-munge-size.patch:1]: Link.

File: sys-cluster/torque/files/torque-6.1.3-port-args.patch [hunk 3]
```diff
@@ -0,0 +1,46 @@
+diff -up torque/contrib/init.d/pbs_mom.in.port-args torque/contrib/init.d/pbs_mom.in
```
Comment (thesamesam) [inline sys-cluster/torque/files/torque-6.1.3-port-args.patch:1]: Link, but is this even relevant for us? Do we use their init script?

File: sys-cluster/torque/files/torque-6.1.3-system-jsoncpp.patch [hunk 4]
```diff
@@ -0,0 +1,294 @@
+diff -up torque/configure.ac.system-jsoncpp torque/configure.ac
```
Comment (thesamesam) [inline sys-cluster/torque/files/torque-6.1.3-system-jsoncpp.patch:1]: Link.

File: sys-cluster/torque/torque-6.1.3.ebuild [hunk 5]
```diff
@@ -5,11 +5,13 @@ EAPI=8
 
 inherit autotools flag-o-matic linux-info toolchain-funcs
```
Comment (thesamesam) [inline sys-cluster/torque/torque-6.1.3.ebuild:6]: I wouldn't `git mv` for this.

File: sys-cluster/torque/files/torque-6.1.3-port-args.patch (hunk 3 above)
Comment (scardracs) [reply sys-cluster/torque/files/torque-6.1.3-port-args.patch:1]: No, we don't

**Pull request closed without merging**